var/spack/repos/spack_repo/builtin/packages/py_codebasin/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack_repo.builtin.build_systems.python import PythonPackage
+
+from spack.package import *
+
+
+class PyCodebasin(PythonPackage):
+    """An analysis tool providing insight into the portability and
+    maintainability of an application's source code."""
+
+    homepage = "https://intel.github.io/code-base-investigator/"
+
+    url = "https://github.com/intel/code-base-investigator/archive/refs/tags/2.0.0.tar.gz"
+
+    maintainers("pennycook")
+
+    license("BSD-3-Clause", checked_by="pennycook")
+
+    version("2.0.0", sha256="f19af5418ab470f1cc22a503f24526ecc7de947f4cdcf897b9c1189d5d97d8e9")
+
+    depends_on("cxx", type="build")
+    depends_on("fortran", type="build")
+
+    depends_on("python@3.12", type=("build", "run"))
```

Comment:
According to `pyproject.toml` for this version `3.12` is the minimum version:

```
requires-python = ">=3.12"
```

So this should be:

```suggestion
    depends_on("python@3.12:", type=("build", "run"))
```

Reply:
Fixed in 44af4156b9a15fcbbc3f1a445f8743a297a58b8b.